mypy/test/testsemanal.py

```diff
@@ -26,18 +24,14 @@
 semanal_files = find_test_files(
     pattern="semanal-*.test",
     exclude=[
-        "semanal-errors-python310.test",
         "semanal-errors.test",
+        "semanal-errors-python310.test",
```

Comment:
```suggestion
        "semanal-errors-python310.test",
        "semanal-errors.test",
```
Probably not worth the effort, but this would remove an unnecessary change. _Not sure the Github suggestion would apply cleanly. I'm not yet fully comfortable with the new files interface._